packages/utils/src/utils/router.js

```diff
@@ -88,6 +88,12 @@ export const paths = {
   serviceAccounts: {
     all() {
       return '/serviceaccounts';
+    },
+    byNamespace() {
```

Comment:
need corresponding change in SideNav.js to handle namespace in the URL, i.e. wrap the url in call to `getPath`
similar issue for trigger resources: https://github.com/tektoncd/dashboard/issues/868